app/navigation/OtherBeneficiaryNavigator.scala

```diff
@@ -34,11 +34,23 @@ class OtherBeneficiaryNavigator @Inject()() extends Navigator {
       yesNoNavigation(draftId)
 
   private def simpleNavigation(draftId: String):  PartialFunction[Page, ReadableUserAnswers => Call] = {
-    case DescriptionPage(index) => _ => other.routes.DiscretionYesNoController.onPageLoad(index, draftId)
+//    case DescriptionPage(index) => _ => other.routes.DiscretionYesNoController.onPageLoad(index, draftId)
```

Comment:
can we get rid of the commented out code or has it been left there for a reason?

Reply:
Updated